src/main/java/com/ivanmostovyi/demo/config/AsyncConfig.java

```diff
@@ -1,55 +1,25 @@
 package com.ivanmostovyi.demo.config;
 
-import com.ivanmostovyi.demo.domain.User;
-import com.ivanmostovyi.demo.dto.MessageFormDto;
 import com.ivanmostovyi.demo.exception.MessageSendingException;
-import com.ivanmostovyi.demo.service.MessageService;
-import com.ivanmostovyi.demo.service.UserService;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
-import org.springframework.boot.CommandLineRunner;
-import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.scheduling.annotation.AsyncConfigurer;
 
+@Slf4j
 @Configuration
-public class AsyncConfig implements AsyncConfigurer, CommandLineRunner {
-
-    private ApplicationContext context;
-
-    private MessageService messageService;
-
-    private UserService userService;
-
-    public AsyncConfig(ApplicationContext context) {
-        this.context = context;
-    }
+public class AsyncConfig implements AsyncConfigurer {
 
     @Override
     public AsyncUncaughtExceptionHandler getAsyncUncaughtExceptionHandler() {
         return (throwable, method, objects) -> {
 
-            if (throwable instanceof MessageSendingException){
```

Comment:
actually, this is needed for handing exceptions from `createInboxMessage `method itself